.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,22 @@
+{
+    "configurations": [
```

Comment:
I have limited understanding of how vscode is working.
Could you explain why we need this?

Reply:
I can remove all the .vscode/ files if you want to.

This particular file is kinda auto generated by the c++ plugin for vscode. Sadly it contains some real settings and some generated ones.

Anyway if you want I can remove all .vscode/ files.

Reply:
Perfect! Yes, I would like not having this file.